roachtest: npgsql failed

roachtest.npgsql [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/11606168?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/11606168?buildTab=artifacts#/npgsql) on release-23.1 @ [58ac87a9b24e1cfb446143dafe04808ea186637f](https://github.com/cockroachdb/cockroach/commits/58ac87a9b24e1cfb446143dafe04808ea186637f):


```
(orm_helpers.go:212).summarizeFailed: 
Tests run on Cockroach v23.1.9-dev-58ac87a9b24e1cfb446143dafe04808ea186637f
Tests run against npgsql v7.0.2
3285 Total Tests Run
2548 tests passed
737 tests failed
248 tests skipped
56 tests ignored
0 tests passed unexpectedly
1 test failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: Npgsql.Tests.CopyTests(NonMultiplexing).Text_import_empty -  (unexpected)
For a full summary look at the npgsql artifacts
An updated blocklist (npgsqlBlocklist) is available in the artifacts' npgsql log
test artifacts and logs in: /artifacts/npgsql/run_1
```
<p>Parameters: <code>ROACHTEST_arch=amd64</code>
, <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/p/roachfana/teamcity-11606168-1693719834-17-n1cpu4/1693749950139/1693750202701)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #109960 roachtest: npgsql failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-master release-blocker]
</p>
</details>
/cc @cockroachdb/sql-foundations
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*npgsql.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>


Jira issue: CRDB-31198

roachtest.npgsql [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/11616147?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/11616147?buildTab=artifacts#/npgsql) on release-23.1 @ [fde98bbb2423906a7e3bfd9cd7956bc3f29918ad](https://github.com/cockroachdb/cockroach/commits/fde98bbb2423906a7e3bfd9cd7956bc3f29918ad):


```
(orm_helpers.go:212).summarizeFailed: 
Tests run on Cockroach v23.1.9-dev-fde98bbb2423906a7e3bfd9cd7956bc3f29918ad
Tests run against npgsql v7.0.2
3285 Total Tests Run
2548 tests passed
737 tests failed
248 tests skipped
56 tests ignored
0 tests passed unexpectedly
1 test failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: Npgsql.Tests.CommandTests(Multiplexing).Cursor_move_RecordsAffected -  (unexpected)
For a full summary look at the npgsql artifacts
An updated blocklist (npgsqlBlocklist) is available in the artifacts' npgsql log
test artifacts and logs in: /artifacts/npgsql/run_1
```
<p>Parameters: <code>ROACHTEST_arch=amd64</code>
, <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/p/roachfana/teamcity-11616147-1693806253-17-n1cpu4/1693832266288/1693832515466)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #109960 roachtest: npgsql failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-master release-blocker]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*npgsql.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.npgsql [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/11626659?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/11626659?buildTab=artifacts#/npgsql) on release-23.1 @ [6e34fffddbb0a88612b712ec5d41e4781d130bab](https://github.com/cockroachdb/cockroach/commits/6e34fffddbb0a88612b712ec5d41e4781d130bab):


```
(orm_helpers.go:212).summarizeFailed: 
Tests run on Cockroach v23.1.9-dev-6e34fffddbb0a88612b712ec5d41e4781d130bab
Tests run against npgsql v7.0.2
3285 Total Tests Run
2548 tests passed
737 tests failed
248 tests skipped
56 tests ignored
0 tests passed unexpectedly
1 test failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: Npgsql.Tests.CopyTests(NonMultiplexing).Text_import_empty -  (unexpected)
For a full summary look at the npgsql artifacts
An updated blocklist (npgsqlBlocklist) is available in the artifacts' npgsql log
test artifacts and logs in: /artifacts/npgsql/run_1
```
<p>Parameters: <code>ROACHTEST_arch=amd64</code>
, <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/p/roachfana/teamcity-11626659-1693892641-17-n1cpu4/1693921757964/1693922005988)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #109960 roachtest: npgsql failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-master release-blocker]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*npgsql.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

--- a/pkg/cmd/roachtest/tests/npgsql_blocklist.go
+++ b/pkg/cmd/roachtest/tests/npgsql_blocklist.go
@@ -745,6 +745,7 @@ var npgsqlBlocklist = blocklist{
 }
 
 var npgsqlIgnoreList = blocklist{
+	`Npgsql.Tests.CommandTests(Multiplexing).Cursor_move_RecordsAffected `:                            "flaky",
 	`Npgsql.Tests.CommandTests(Multiplexing).QueryNonQuery`:                                           "flaky",
 	`Npgsql.Tests.CommandTests(Multiplexing).SingleNonQuery`:                                          "flaky",
 	`Npgsql.Tests.CommandTests(Multiplexing).Statement_mapped_output_parameters(Default)`:             "flaky",
@@ -786,6 +787,7 @@ var npgsqlIgnoreList = blocklist{
 	`Npgsql.Tests.CopyTests(NonMultiplexing).Import_string_array`:                                     "flaky",
 	`Npgsql.Tests.CopyTests(NonMultiplexing).Import_string_with_buffer_length`:                        "flaky",
 	`Npgsql.Tests.CopyTests(NonMultiplexing).Prepended_messages`:                                      "flaky",
+	`Npgsql.Tests.CopyTests(NonMultiplexing).Text_import_empty`:                                       "flaky",
 	`Npgsql.Tests.CopyTests(NonMultiplexing).Undefined_table_throws`:                                  "flaky",
 	`Npgsql.Tests.CopyTests(NonMultiplexing).Write_column_out_of_bounds_throws`:                       "flaky",
 	`Npgsql.Tests.CopyTests(NonMultiplexing).Write_null_values`:                                       "flaky",

--- a/pkg/cmd/roachtest/tests/pgjdbc_blocklist.go
+++ b/pkg/cmd/roachtest/tests/pgjdbc_blocklist.go
@@ -924,6 +924,7 @@ var pgjdbcIgnoreList = blocklist{
 	"org.postgresql.test.jdbc2.BatchedInsertReWriteEnabledTest.testReWriteDisabledForPlainBatch[3: autoCommit=NO, binary=FORCE]":                    "54477",
 	"org.postgresql.test.jdbc2.CursorFetchTest.testBasicFetch[binary = FORCE]":                                                                      "flaky",
 	"org.postgresql.test.jdbc2.CursorFetchTest.testBasicFetch[binary = REGULAR]":                                                                    "flaky",
+	"org.postgresql.test.jdbc2.CursorFetchTest.testMultiRowResultPositioning[binary = FORCE]":                                                       "flaky",
 	"org.postgresql.test.jdbc2.DatabaseEncodingTest.testBadUTF8Decode":                                                                              "54477",
 	"org.postgresql.test.jdbc2.DatabaseEncodingTest.testTruncatedUTF8Decode":                                                                        "54477",
 	"org.postgresql.test.jdbc2.DatabaseEncodingTest.testUTF8Decode":                                                                                 "54477",

